src/client.cpp

```diff
@@ -57,7 +57,9 @@ namespace oxen::quic
 
```

Comment:
The line above is doing nothing (`context` is already a `shared_ptr<ClientContext>`).  Further, the line just below here is using this `ctx` but the bit added in this commit just below that is using `context` anyway.

Reply:
Yup you’re exactly right, it does nothing. That’s leftover from when it was casting a reference to the base class of client and server contexts. Was trying to get this PR up before flying out, completely missed that. Thanks for catching that 👍🏼